### PR TITLE
build(deps): upgrade conventional-commit-types to v3.0.0

### PR DESCRIPTION
**Pull request by felipecassiors**
It dropped `improvements` type.

Changelog:
https://github.com/commitizen/conventional-commit-types/compare/v2.3.0...v3.0.0

Follow-up: https://github.com/zeke/semantic-pull-requests/pull/94 https://github.com/zeke/semantic-pull-requests/pull/96

### CODE REVIEW

**Comment by felipecassiors** [comment]
The tests didn't run. 😥

**Comment by felipecassiors** [comment]
Now they run! 😀

**Comment by felipecassiors** [comment]
@zeke how about the last commit?

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: ci
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```
Comment (zeke) [inline .github/workflows/ci.yml:10]:
```suggestion
on: push
```

**Comment by zeke** [comment]
Let's try it again! 🤞

**Comment by felipecassiors** [comment]
@zeke but the way you did, we lose tests on pull requests.

**Comment by zeke** [comment]
`on: push` should run the action for all pushes, whether they're in a PR or not, right?

**Comment by felipecassiors** [comment]
Not really. The `on: push` runs on every push for every branch in your repository (if it is a fork, it doesn't run). 

The `on: pull_request` runs on every pull request despite it is from a fork or not, and also post the status in the Pull Request checks.

**Pull request merged**